dae/dae/variants_loaders/vcf/tests/test_vcf_loader.py

```diff
@@ -221,6 +170,40 @@ def test_multivcf_loader_fill_missing(
 #     )
 
 
+@pytest.fixture
+def inheritance_trio_denovo_omission_ped(
+    tmp_path_factory: pytest.TempPathFactory
+) -> Path:
+    root_path = tmp_path_factory.mktemp("inheritance_trio_denovo_omission_ped")
+    ped_path = setup_pedigree(root_path / "ped_data" / "in.ped", """
+    familyId	personId	dadId	momId	sex	status	role
+    f1		    mom1		0	    0	    2	1	    mom
+    f1		    dad1		0	    0	    1	1	    dad
+    f1		    ch1		    dad1	mom1	2	2	    prb
+    """)
+
+    return ped_path
+
+
+@pytest.fixture
+def inheritance_trio_denovo_omission_vcf(
+    tmp_path_factory: pytest.TempPathFactory
+) -> Path:
+    root_path = tmp_path_factory.mktemp("inheritance_trio_denovo_omission_vcf")
+    vcf_path = setup_vcf(root_path / "vcf_data" / "in.vcf", """
+    ##fileformat=VCFv4.2
+    ##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+    ##INFO=<ID=INH,Number=1,Type=String,Description="Inheritance">
+    ##contig=<ID=1>
+    #CHROM	POS	    ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	mom1	dad1	ch1
+    1	    11515	.	T	G	.	    .   	INH=OMI	GT	    0/0 	1/0 	1/1
+    1	    11523	.	T	G	.	    .   	INH=DEN	GT	    1/1 	1/1 	1/0
+    1	    11524	.	T	G	.	    .   	INH=DEN	GT	    1/1 	1/1 	0/1
+    """)
+
+    return vcf_path
+
+
```

Comment:
consider merging this fixtures into a single fixture, that returns tuple: `(str(ped_path), str(vcf_path))`